src/acpi_types.rs

```diff
@@ -131,10 +135,11 @@ impl fmt::Debug for MemoryAffinity {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(
             f,
-            "MemoryAffinity {{ {:#x} -- {:#x}, node#{} }}",
+            "MemoryAffinity {{ {:#x} -- {:#x}, node#{}, Non-volatile: {} }}",
             self.start(),
             self.end(),
-            self.proximity_domain
+            self.proximity_domain,
+            self.is_non_volatile()
```

Comment:
maybe do `if self._is_non_volatile() "nvm" else ""` so debug output only shows nvm in case it is?

Reply:
Done